homeassistant/components/nut/const.py

```diff
@@ -453,6 +453,12 @@
         device_class=DEVICE_CLASS_TEMPERATURE,
         state_class=STATE_CLASS_MEASUREMENT,
     ),
+    "watts": SensorEntityDescription(
```

Comment:
`watts` doesn't seem to be listed as a variable here https://networkupstools.org/docs/user-manual.chunked/apcs01.html

Where did you find this value?

Reply:
I have the NUT to influx exporter and it come through as a parameter. Looks like that document hasn’t been updated since 2016. Might be a new measurement?